src/authorization/tests/handler.test.js

```diff
@@ -0,0 +1,59 @@
+const { createHash, encryptData } = require('../../crypto/crypto');
+const { addUserInDB } = require('../handler.addUserInDB')
+const PersonController = require('../../PersonCRUD/person.controller')
+
+const data = {
+    name: "Andriy",
+    email: "hasherama8@gmail.com",
+    password: "password"
+}
+const testPasswordCipher = '123456';
+const testPasswordHash = 'qwerty'
+
+jest.mock('../../crypto/crypto', () => {
+    const originalModule = jest.requireActual('../../crypto/crypto');
+
+    return {
+        __esModule: true,
+        ...originalModule,
+        createHash: jest.fn(() => testPasswordCipher),
+        encryptData: jest.fn(() => testPasswordHash)
+    };
+});
+
+jest.mock('../../PersonCRUD/person.controller', () => {
+    const originalModule = jest.requireActual('../../PersonCRUD/person.controller');
+
+    return {
+        __esModule: true,
+        ...originalModule,
+        createPerson: jest.fn(() => 'test add in DB was successful')
+    };
+});
+
+beforeAll(async () => {
+    await addUserInDB(data)
```

Comment:
You should call tested function in the `test`, not in `beforeAll`. You have good example in RAM/tests/ram.test.js file